public/data/sanjay-kv.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Sanjay KV",
+  "bio": "Founder of Recode Hive",
+  "avatar": "https://avatars.githubusercontent.com/u/30715153?v=4",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sanjay-kv",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sanjay_k_v",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn more about Open Source on my YouTube channel",
+      "url": "https://www.youtube.com/channel/UCzyGIdENFVT36Yx4zTws4kw",
+      "icon": "youtube"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Content Creation",
+      "date": "May 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Founder of Recode Hive",
+      "url": "https://recodehive.com/"
+    },
+    {
+      "title": "Started YouTube",
+      "image": "http://github.com/eddiejaoude.png",
+      "date": "June 2019",
+      "icon": "youtube",
+      "color": "red",
+      "description": "First real video",
+      "url": "http://youtube.com/eddiejaoude"
+    },
+    {
+      "title": "GitHub Star",
+      "date": "2020",
+      "icon": "github",
+      "color": "green",
+      "description": "Became a GitHub Star with 30 other people",
+      "url": "http://github.com/eddiejaoude"
+    },
+    {
+      "title": "GitHub Star of the Year",
+      "date": "2020",
+      "icon": "github",
+      "color": "green",
+      "description": "Won GitHub Star of the Year out of 55+ million people",
+      "url": "http://github.com/eddiejaoude"
```

Comment:
This is not your data?

Reply:
oops. thats a mistake. i forgot to add that changes to commit